### ISSUE TITLE
PipeWire: Volume is not retained after new pipewire session

### ISSUE THREAD

**Issue by the-eater** [clangen/musikcube#441]
When you set the volume of musikcube to e.g. 85%, then reboot the system, and start musikcube again, the volume will still show 85% but in reality is playing at a lower volume, changing the volume jumps it to the correct value

**Comment by clangen** [comment]
Weird -- I tried to reproduce this issue and couldn't. I did find an edge case where this can happen, but I really doubt it's your problem. You can pull `master` and try again, but I'd be surprised if it fixed your issue. What distro and version of PipeWire are you running?

**Comment by the-eater** [comment]
hm doesn't fix the issue no

I am using Void Linux with pipewire 0.3.31

**Comment by the-eater** [comment]
oh ho!

guess what I found in my `restore-stream` file

```json
 "restore.stream.Output/Audio.application.name:musikcube": { "volume": 1.000000, "mute": false, "volumes": [ 0.300001, 0.300001 ], "channels": [ "FL", "FR" ] },
 ```

**Comment by the-eater** [comment]
yep that fixed it, so if you play musikcube via PulseAudio it will make the restore.stream entry and apparently enforce this until you change volume?
